src/components/error-pages/error-browser-not-supported/error-browser-not-supported.ts

```diff
@@ -48,10 +46,6 @@ export class MErrorBrowserNotSupported extends ModulVue {
 
 const ErrorBrowserNotSupported: PluginObject<any> = {
     install(v, options): void {
-        v.prototype.$log.debug(ERROR_BROWSER_NOT_SUPPORTED_NAME, 'plugin.install');
-        v.use(I18nPlugin);
-        v.use(LinkPlugin);
-        v.use(MediaQueriesPlugin);
```

Comment:
En enlevant MediaQueriesPlugin ça va encore marcher dans ce cas par contre ça enlèvera la ref à 
(v.prototype as any).$mq = mediaQueries; (qui se trouve dans media-queries.ts)
On est ok avec ça?

Reply:
pis en plus dans mediaQueriesMixin on a : 

public get isMqMinXL(): boolean {
        return this.$mq.state.isMqMinXL;
    }

Des références à $mq donc là je comprend vraiment pas comment ça peut marcher sans l'usage du plugin >.<

Reply:
MediaQueriesPlugin  est un plugin globale qui est installé au démarage par UtilsPlugin